sandy/sections/mf8.py

```diff
@@ -248,6 +248,11 @@ def _read_rdd(tape, mat):
     'DRICK': 0.0,
     'RICL': 0.0,
     'DRICL': 0.0}
+
```

Comment:
can you put back the previous test? that test would have failed in your previous commit...so, it is better to keep it!

Reply:
I did a test with the previous example and I did not find errors, anyway I rechanged it

Reply:
But now, why did you delete the other test?